backend/shared_library/utils/singleton.py

```diff
@@ -19,7 +19,8 @@ class Singleton:
 
     _instance = None
 
-    def __new__(class_, *args, **kwargs):
+    # TODO: remove arg1. Currently I'm not sure how to handle this
```

Comment:
Why is a todo there? shouldn't the code be done lol

Reply:
`*args` and `**kwargs` could be removed in future, but I dont know how